kong/clustering/compat/init.lua

```diff
@@ -360,6 +360,26 @@ function _M.update_compatible_payload(payload, dp_version, log_suffix)
     end
   end
 
+  if dp_version_num < 3003000000 --[[ 3.3.0.0 ]] then
+    -- remove updated_at field for core entities ca_certificates, certificates, consumers,
+    -- targets, upstreams, plugins, workspaces, clustering_data_planes and snis
+    local entity_names = {'ca_certificates', 'certificates', 'consumers', 'targets', 'upstreams',
+      'plugins', 'workspaces', 'clustering_data_planes', 'snis'}
+
+    for _, name in ipairs(entity_names) do
+      local config_entities = config_table[name]
+      if config_entities then
+        for _, config_entity in ipairs(config_entities) do
```

Comment:
We could use `for _, config_entity in ipairs(config_entities) or {} do`,
then remove `if config_entities then end` statement.